npm/webpack-preprocessor/package.json

```diff
@@ -46,7 +50,7 @@
     "eslint-plugin-mocha": "8.1.0",
     "fast-glob": "3.1.1",
     "find-webpack": "1.5.0",
-    "fs-extra": "9.1.0",
+    "md5": "2.3.0",
```

Comment:
Shouldn’t this be a production dependency? See https://github.com/cypress-io/cypress/issues/24051.

Reply:
Yes indeed! Thanks for bringing this to my attention as I had missed the issue for it. Will be fixed by https://github.com/cypress-io/cypress/pull/24098.